README.md

```diff
@@ -4,7 +4,38 @@ React hooks for fetching data from gtfs sources.
 
 ## Usage
 
-TODO: Describe.
+There are two hooks which this package provides. `useGtfsSchedule` and `useGtfsRealtime`. Both take two arguments, a resolver and a timeout.
+
+The resolver must be an async function which takes no arguments and returns an array buffer containing the raw data for the hook to process.
```

Comment:
```suggestion
The resolver must be an async function which takes no arguments and returns a `Uint8Array` containing the raw data for the hook to process.
```

`gtfs-realtime-bindings` specifically asks for this type both in its tsc definition and its jsdoc.

Even if it does technically support others, I think it's best to specify this type for both hooks (seems both simpler and safer)?